build/yaml/crd/nsx.vmware.com_vpcnetworkconfigurations.yaml

```diff
@@ -94,6 +94,9 @@ spec:
                   context in logs. Less than or equal to 8 characters.
                 maxLength: 8
                 type: string
+              vpcPath:
+                description: Policy path of the NSX VPC the Namespace associated with.
```

Comment:
In the description, we also need to make it clear that if vpcPath is provided, then other fields except defaultSubnetSize， podSubnetAccessMode do not take effect. 
could you also add a sample under build/yaml/samples?

Reply:
Updated.